.github/workflows/xpu-pr.yml

```
@@ -0,0 +1,384 @@
+  # Currently we only have 2-XPU runners, so this job runs the gpu_2 marker.
+  # TODO: when 4-XPU XPU runners are available, change min_gpus to 4 and the
+  #       marker to 'pre_merge and vllm and (gpu_2 or gpu_4)' and use the
+  #       runner label `[self-hosted, xpu, multi-gpu-4]`.
+  vllm-xpu-multi-gpu-tests:
+    name: vllm-xpu-runtime
+    needs: [changed-files, vllm-xpu-build, vllm-xpu-sanity]
+    if: needs.changed-files.outputs.core == 'true' || needs.changed-files.outputs.vllm == 'true'
+    # TODO: use [self-hosted, xpu, multi-gpu] when multiple runner pools
+    # exist (e.g. 1-GPU pool vs 4-GPU pool). For now, all XPU jobs use the
+    # same runner and setup-xpu's min_gpus check gates GPU count at runtime.
+    runs-on: [self-hosted, xpu]
```

actionlint (1.7.12)

[error] label "xpu" is unknown. available labels are "windows-latest", "windows-latest-8-cores", "windows-2025", "windows-2025-vs2026", "windows-2022", "windows-11-arm", "ubuntu-slim", "ubuntu-latest", "ubuntu-latest-4-cores", "ubuntu-latest-8-cores", "ubuntu-latest-16-cores", "ubuntu-24.04", "ubuntu-24.04-arm", "ubuntu-22.04", "ubuntu-22.04-arm", "macos-latest", "macos-latest-xlarge", "macos-latest-large", "macos-26-intel", "macos-26-xlarge", "macos-26-large", "macos-26", "macos-15-intel", "macos-15-xlarge", "macos-15-large", "macos-15", "macos-14-xlarge", "macos-14-large", "macos-14", "self-hosted", "x64", "arm", "arm64", "linux", "macos", "windows". if it is a custom label for self-hosted runner, set list of labels in actionlint.yaml config file

(runner-label)